sqlmesh/core/scheduler.py

```diff
@@ -116,6 +117,11 @@ def evaluate(
             snapshot.name: snapshot,
         }
 
+        if isinstance(snapshot.model, SeedModel) and not snapshot.model.is_hydrated:
```

Comment:
it may happen if the error occurred during the plan application and we created a record, but never actually evaluated the snapshot itself. I don't think we should raise in this case. It should be a rare occasion.